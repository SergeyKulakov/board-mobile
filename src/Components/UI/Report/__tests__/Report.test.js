import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { Report as Component } from '../index'

describe('Report', () => {
  const defaultProps = {
    onClick: () => null,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
