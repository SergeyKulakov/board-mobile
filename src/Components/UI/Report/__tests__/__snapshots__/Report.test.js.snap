// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Report render snapshot 1`] = `
<TouchableWithoutFeedback
  onPress={[Function]}
>
  <Styled(View)>
    <Styled(Icon) />
    <Styled(Text)>
      Report
       
    </Styled(Text)>
  </Styled(View)>
</TouchableWithoutFeedback>
`;
